pyaedt/modules/Material.py

```diff
@@ -213,52 +211,274 @@ def set_type(self, type):
         -------
 
         """
-        self.type = type
-        if self.type == "simple":
-            self.property_value = [BasicValue()]
-        elif self.type == "anisotropic":
-            self.property_value = [BasicValue() for i in range(3)]
-        elif self.type == "tensor":
-            self.property_value = [BasicValue() for i in range(9)]
-        elif self.type == "nonlinear":
-            self.property_value = [BasicValue()]
+        self._type = type
+        if self._type == "simple":
+            self._property_value = [self._property_value[0]]
+        elif self._type == "anisotropic":
+            self._property_value = [self._property_value[0] for i in range(3)]
+        elif self._type == "tensor":
+            self._property_value = [self._property_value[0] for i in range(9)]
+        elif self._type == "nonlinear":
+            self._property_value = [self._property_value[0]]
 
+    @property
+    def value(self):
+        if len(self._property_value) == 1:
+            return self._property_value[0].value
+        else:
+            return [i.value for i in self._property_value]
+
+    @value.setter
+    def value(self, val):
+        if isinstance(val, list):
+            i = 0
+            for el in val:
+                if i >= len(self._property_value):
+                    self._property_value.append(BasicValue())
+
+                self._property_value[i].value = el
+                i += 1
+        else:
+            self._property_value[0].value = val
 
+    @property
+    def unit(self):
```

Comment:
We've got to get in the habit of providing docstrings for every method (and property).  Doc team has a hard time as is, and it's generally unclear (even for me) what these methods are doing.